src/Pusher.php

```diff
@@ -94,9 +95,15 @@ public function __construct($auth_key, $secret, $app_id, $options = array(), $ho
 
         /* End backward compatibility with old constructor **/
 
+        $useTLS = false;
+        if (isset($options['useTLS'])) {
+            $useTLS = $options['useTLS'] === true;
+        } elseif (isset($options['encrypted'])) {
+            // `encrypted` deprecated in favor of `forceTLS`
+            $useTLS = $options['encrypted'] === true;
+        }
         if (
-            isset($options['encrypted']) &&
-            $options['encrypted'] === true &&
+            $useTLS &&
```

Comment:
Does this logic work for sure when you don't set either option?

Reply:
If you don't set either, then $useTLS will remain the default value of `false`. This is the value table:

```
useTLS | encrypted | $useTLS
-------|-----------|--------
unset  | unset     | false
false  | unset     | false
true   | unset     | true
unset  | false     | false
false  | false     | false
true   | false     | true
unset  | true      | true
false  | true      | false
true   | true      | true
```